apps/pie-docs/src/patterns/overlay-patterns/overview/overview.md

```diff
@@ -0,0 +1,47 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: 'Overlay Patterns'
+    order: 1
+shouldShowContents: true
+permalink: patterns/overlay-patterns/
+---
+
+
+## What are overlay patterns?
+
+Overlay patterns in UI design involve the presentation of additional content or functionality on top of existing elements within the user interface. These overlays can be temporary or permanent and serve various purposes, such as providing contextually relevant information, enabling interactions, or offering supplementary options.
+
+By overlaying content, designers can optimise screen real estate and enhance user experience by reducing the need for users to navigate away from the current context.
+
+{% contentPageImage {
+  src:"../../../assets/img/patterns/haptic-feedback/overview-desktop.svg",
```

Comment:
out of interest, why are we using a haptics image to introduce overlays? It may make even less sense listening to the `img` alt description

Reply:
We have agreed to move on implementing this page using the placeholder illustration that will be updated as soon as the Overlay illustration is ready.

Reply:
This image has been updated according to a new design.